packages/cozy-harvest-lib/src/helpers/triggers.js

```diff
@@ -0,0 +1,85 @@
+import { randomDayTime } from './daytime'
+
+const DAILY = 'daily'
+const HOURLY = 'hourly'
+const MONTHLY = 'monthly'
+const WEEKLY = 'weekly'
+const VALID_FREQUENCIES = [DAILY, HOURLY, MONTHLY, WEEKLY]
+
+const DEFAULT_FREQUENCY = WEEKLY
+const DEFAULT_CRON = '0 0 0 * * 0' // Once a week, sunday at midnight
+// By default konnectors are run at random hour between 00:00PM and 05:00AM.
+const DEFAULT_TIME_INTERVAL = [0, 5]
+
+/**
+ * Stategies for building cron values
+ * See https://docs.cozy.io/en/cozy-stack/jobs/#cron-syntax
+ */
+const getCronStrategies = options => {
+  const { dayOfMonth, dayOfWeek, hours, minutes } = options
+  return {
+    [DAILY]: `0 ${minutes} ${hours} * * *`,
+    [HOURLY]: `0 ${minutes} * * * *`,
+    [MONTHLY]: `0 ${minutes} ${hours} ${dayOfMonth} * *`,
+    [WEEKLY]: `0 ${minutes} ${hours} * * ${dayOfWeek}`
+  }
```

Comment:
Here I think we should return the only one strategy available according to provided options. What happens if I provide only hours and minutes but I use the weekly strategy? Or at least the unprovided should have default values.